src/test/java/com/google/api/generator/gapic/composer/ComposerTest.java

```diff
@@ -65,58 +65,90 @@ public void gapicClass_addApacheLicense() {
 
   @Test
   public void composeSamples_showcase() {
-    for (Sample sample : samples) {
-      assertEquals(
-          "File header will be empty before composing samples",
-          sample.fileHeader(),
-          ImmutableList.of());
-      assertEquals(
-          "ApiShortName will be empty before composing samples",
-          sample.regionTag().apiShortName(),
-          "");
-      assertEquals(
-          "ApiVersion will be empty before composing samples", sample.regionTag().apiVersion(), "");
-    }
+    GapicClass testClass = clazzes.get(0).withSamples(ListofSamples);
+    List<GapicClass> testClassList = Arrays.asList(new GapicClass[] {testClass});
 
     List<Sample> composedSamples =
-        Composer.prepareExecutableSamples(clazzes, protoPackage).get(0).samples();
+        Composer.prepareExecutableSamples(testClassList, protoPackage).get(0).samples();
 
+    assertFalse(composedSamples.isEmpty());
     for (Sample sample : composedSamples) {
       assertEquals(
-          "File header should be apache",
-          sample.fileHeader(),
-          Arrays.asList(CommentComposer.APACHE_LICENSE_COMMENT));
-      assertEquals(
-          "ApiShortName should be showcase", sample.regionTag().apiShortName(), "showcase");
-      assertEquals("ApiVersion should be v1beta1", sample.regionTag().apiVersion(), "v1beta1");
+          "File header should be APACHE",
+          Arrays.asList(CommentComposer.APACHE_LICENSE_COMMENT),
+          sample.fileHeader());
+      assertEquals("ApiShortName should be empty", "", sample.regionTag().apiShortName());
+      assertEquals("ApiVersion should be V1beta1", "V1Beta1", sample.regionTag().apiVersion());
     }
   }
 
+  @Test
+  public void parseDefaultHostTest() {
```

Comment:
We usually follow this [pattern](go/java-style#s5.2.3-method-names) for unit test names

Reply:
Thanks; updated.